### ISSUE TITLE
Clear cache on updating to a new version

### ISSUE THREAD

**Issue by dariusf** [HubTurbo/HubTurbo#942]
We currently don't have a way to change the schema for existing users in a robust way. Until we do, a stop-gap measure is to clear the cache on updating to a new version.

We should probably also come up with something for settings, which we can't simply clear.

**Comment by damithc** [comment]
Another option is to use the major version number to indicate backward compatibility. Any change in the major version number can be taken as a signal to re-download the files. That's also in line with the 'semantic versioning' convention.

**Comment by dariusf** [comment]
Yup, we should do that eventually. Currently schema changes happen pretty frequently though.

It will also require us to modify the updater. The main app doesn't know about the previous version number, only that it was just updated, from command line flags that the updater starts it with.
